creator-node/src/config.js

```diff
@@ -835,10 +835,10 @@ const config = convict({
     env: 'reconfigSPIdBlacklistString',
     default: '1,4,33,37,39,40,41,42,43,52,56,58,59,60,61,64,65'
   },
-  syncOverridePassword: {
-    doc: 'Used to allow manual syncs to be issued on foundation nodes only, and still requires password',
+  overridePassword: {
```

Comment:
are there any backwards compatibility issues?

Reply:
nothing significant - just need to change envvar on nodes. this is only for manual use so no concern